tool/modules/uds_fuzz.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python
+# Released under GNU General Public License v3
+# https://github.com/Cr0wTom/UDS-Seed-Randomness-Fuzzer
+
+from __future__ import print_function
+from lib.can_actions import auto_blacklist
+from lib.common import list_to_hex_str, parse_int_dec_or_hex, str_to_int_list
+from lib.constants import ARBITRATION_ID_MAX, ARBITRATION_ID_MAX_EXTENDED
+from lib.constants import ARBITRATION_ID_MIN
+from lib.iso15765_2 import IsoTp
+from lib.iso14229_1 import Constants, Iso14229_1, NegativeResponseCodes, Services
+from sys import stdout, version_info
+import argparse
+import datetime
+import time
+
+
+# Python 2/3 compatibility
+if version_info[0] == 2:
+    range = xrange
+    input = raw_input
+
+NRC_NAMES = {
```

Comment:
This should be imported from the UDS module, rather than copy-pasted into this module.